automerge/README.md

```diff
@@ -0,0 +1,135 @@
+
+# Setup - both cases
+
+Our samples currently work with GitHub. They use the `hub` CLI to create a merge commit that when pushed, will merge the associated Pull Request. See [here](https://hub.github.com/hub-merge.1.html) for more details.
+
+## gitconfig
+
+Since we are creating git commits within the Tekton tasks, we need to know the username and email address to associate with them, as in our ['tekton-example'](../tekton-example/README.md). So, edit `gitconfig` and
+
+```sh
+kubectl create configmap promoteconfigmap --from-file=gitconfig
+```
+
+## Dockerfile
+
+The 'hub' CLI runs in a Docker container within a Tekton Pipeline. We've provided a sample Dockerfile. If you are taking the 'webhooks' path you can comment out the section that installs `yq`. Then, 
+
+```sh
+docker login
+docker build -t YOUR_DOCKER_HUB_ID/hub-test .
+docker push YOUR_DOCKER_HUB_ID/hub-test
+```
+
+## Create a Pull Request
+
+1. Fork our example gitops repository, https://github.com/rhd-gitops-example/gitops-example-dev. 
+1. Create a 'promotion' Pull Request either manually or using our ['tekton-example'](../tekton-example/README.md). For example you can use code of the form, 
+
+```sh
+services promote --from promote-demo --to https://github.com/YOUR_GITHUB_ID/gitops-example-dev.git --service promote-demo
+```
+
+## Generate a GitHub token
+
+You will need a GitHub token with repository access in order to merge Pull Requests. 
+
+## Standalone case
+
+Edit the four files in the 'templates' directory.
+
+- In `automerge-task.yaml` replace `YOUR_DOCKER_HUB_ID` with your DockerHub id.
+- In `git-resource.yaml` replace `YOUR_GITHUB_ID` with your GitHub id.
+- In `github-secret.yaml` replace `[your-github-token-with-repo-access]` with your base64-encoded token. Run this command to get the right string:
+
+```sh
+echo -n [your-token] | base64
+```
+
+- In pull-request.yaml replace YOUR_PULL_REQUEST_URL with your pull request URL, e.g. `https://github.com/mnuttall/gitops-example-dev/pull/22`
+
+Apply all the Tekton resources:
+
+```sh
+kubectl apply -f standalone/resources
+kubectl apply -f standalone/templates
+```
+
+Finally start the Tekton pipeline:
+
+```sh
+tkn pipeline start automerge-pipeline -r source-repo=gitops-repo -r pr=pull-request -p github-config=promoteconfigmap -p github-secret=github-secret --showlog
+```
+
+The pipeline will do a dry run to test that the yaml in the Pull Request is good, then merge the Pull Request and delete the branch associated with it.
+
+## Tekton Dashboard Webhooks Extension
+
+See the [Getting Started](https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/GettingStarted.md) guide for setup guidelines. Our example uses GitHub Enterprise (GHE) and expects webhooks to be delivered to a cluster that is routeable to from GHE.
```

Comment:
nit: "routeable to from GHE" or just "routeable from GHE"

Reply:
deleted 'to'.